src/main/java/net/minecraftforge/client/model/geometry/IGeometryBakingContext.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.model.geometry;
+
+import com.mojang.math.Transformation;
+import net.minecraft.client.renderer.block.model.ItemTransforms;
+import net.minecraft.client.resources.model.Material;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.NamedRenderTypeManager;
+import net.minecraftforge.client.RenderTypeGroup;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * The context in which a geometry is being baked, providing information such as lighting and
+ * {@link ItemTransforms transforms}, and allowing the user to create {@link Material materials} and query
+ * {@link RenderTypeGroup render types}.
+ *
+ * @see StandaloneGeometryBakingContext
+ * @see BlockGeometryBakingContext
+ */
+public interface IGeometryBakingContext
+{
+    /**
+     * Gets the name of the model being baked for logging and caching purposes.
```

Comment:
The javadocs for these getters should use the inline `{@return}` tag.
```suggestion
     * {@return the name of the model being baked for logging and caching purposes}
```